tests/k8st/tests/test_bgp_advert.py

```diff
@@ -263,10 +273,21 @@ def assert_ecmp_routes(self, dst, via=["10.192.0.3", "10.192.0.4"]):
             matchStr += "\n\tnexthop via %s  dev eth0 weight 1" % ip
         retry_until_success(lambda: self.assertIn(matchStr, self.get_routes()))
 
+    def get_svc_host_ip(self, svc, ns):
+        return kubectl("get po -l app=%s -n %s -o json | jq -r .items[0].status.hostIP" %
+                       (svc, ns)).strip()
+
+    def add_svc_external_ips(self, svc, ns, ips):
+        ipsStr = ','.join('"{0}"'.format(ip) for ip in ips)
+        patchStr = "{\"spec\": {\"externalIPs\": [%s]}}" % (ipsStr)
+        return kubectl("patch svc %s -n %s --patch '%s'" % (svc, ns, patchStr)).strip()
+
     def test_rr(self):
         self.tearDown()
         self.setUpRR()
 
+        localSvcExternalIP = "8.8.8.8"
```

Comment:
I'd recommend using something other than `8.8.8.8`, since that's a commonly used DNS server.